lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -16,6 +16,9 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   alias Membrane.RTC.Engine.Endpoint.WebRTC.SimulcastConfig
 
   require Membrane.Logger
+  require Membrane.TelemetryMetrics
+
+  @track_event [Membrane.RTC.Engine, :track]
```

Comment:
```suggestion
  @track_metadata_event [Membrane.RTC.Engine, :track, :metadata, :event]
```
and let's move this to a separate module called `Membrane.RTC.Engine.Endpoint.WebRTC.Metrics`

Reply:
I think, that creating separate module with only one metric definition is not a good idea :(